src/Call/PaymentStatusRequest.php

```diff
@@ -62,8 +67,15 @@ public function send(ApiClient $apiClient): PaymentResponse
 			new ResultCode($data['resultCode']),
 			$data['resultMessage'],
 			isset($data['paymentStatus']) ? new PaymentStatus($data['paymentStatus']) : null,
-			$data['authCode'] ?? null
+			$data['authCode'] ?? null,
+			null,
+			$response->getExtensions()
 		);
 	}
 
+	public function registerExtension(string $name, ResponseExtensionHandler $extensionHandler)
```

Comment:
Nevíme :) `maskClnRP` je jen tady. Ale do ostatních bych to přidával až když to bude potřeba, právě proto že nevíme kdy a co umí